Stratis.Bitcoin/Utilities/NetworkExtensions.cs

```diff
@@ -10,18 +10,6 @@ public static class NetworkExtensions
         /// <summary>Fake height value used in Coins to signify they are only in the memory pool (since 0.8).</summary>
         public const int MempoolHeight = 0x7FFFFFFF;
 
-        /// <summary>Default minimum transaction fee for bitcoin network.</summary>
```

Comment:
so where are all these actually set?

Reply:
The defaults are now in NStratis in Network when the network is created. This avoids the ugly if (bitcoin) logic.

Reply:
ah is it in your latest change to NStratis?

Reply:
Yes it is. It's already merged into fullnode. https://github.com/stratisproject/NStratis/commit/d897a44114b564e982fb03b3408457725cdb055d

Reply:
ah ok cool, I'm happy with this.
Can you please just reply to my other comments and I'll push it in.

Reply:
There you go!